### PR TITLE
[docs] Add 9.0 release notes placeholders

### PR DESCRIPTION
**Pull request by brianseeders**

### CODE REVIEW

**Comment by leemthompo** [comment]
@brianseeders 

> INFO:build_docs:  /tmp/docsbuild/target_repo/html/en/enterprise-search/master/release-notes-8.4.2.html contains broken links to:

I've got a fix for that one at least: https://github.com/elastic/enterprise-search-pubs/pull/4673

**Comment by leemthompo** [comment]
run docs-build

File: docs/reference/release-notes/highlights.asciidoc [hunk 1]
```diff
@@ -9,36 +8,13 @@ ifeval::["{release-state}"!="unreleased"]
...
 endif::[]
 
-// The notable-highlights tag marks entries that
-// should be featured in the Stack Installation and Upgrade Guide:
 // tag::notable-highlights[]
 
 [discrete]
 [[esql_inlinestats]]
-== ESQL: INLINESTATS
+=== ESQL: INLINESTATS
```
Comment (jakelandis) [inline docs/reference/release-notes/highlights.asciidoc:17]: I think these release highlights are relevant to 8.x, no 9.0. However, I assume there will be few enough of these to allow for manual clean up in a followup.

File: docs/reference/release-notes/highlights.asciidoc [hunk 2]
```diff
@@ -1,6 +1,5 @@
-[chapter]
```
Comment (leemthompo) [inline docs/reference/release-notes/highlights.asciidoc:1]: @brianseeders we recently moved some of these files around in the docs nav so we need to keep the `[chapter]` id going forward and the title should start at H1

**Comment by brianseeders** [comment]
Closing this so that others can handle, I don't need to be a blocker for this

**Pull request closed without merging**